tasks/lib/compass.js

```diff
@@ -46,7 +46,7 @@ exports.init = function (grunt) {
       if (supportedOptions.indexOf(underscoredOption) >= 0) {
         // naively escape double-quotes in the value
         var value = options[option].replace(/"/g, '\\"');
```

Comment:
I changed this and also the `extractRawOptions` test to check if single quotes escaped correctly.

I also notice that you use double quotes all over the test file. Should I also test the single quotes on every single one?